src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentOrderService.java

```diff
@@ -58,6 +59,34 @@ public List<String> performCheck(CallbackRequest callbackRequest, String userAut
         return errors;
     }
 
+    public List<String> performCheck(FinremCallbackRequest callbackRequest, String userAuthorisation, FinremCaseDetailsMapper finremCaseDetailsMapper) {
```

Comment:
finremCaseDetailsMapper - you shouldn't need to pass a dependency as an argument. Just make it a dependency of ConsentOrderService